ossdbtoolsservice/language/completion/mysql_completion.py

```diff
@@ -0,0 +1,20 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from prompt_toolkit.completion import Completion    # noqa
+
```

Comment:
remove extra line

Reply:
Which line? The standard in our codebase seems to be one line empty lines before imports and two before the class.